serverlessworkflow/modules/ROOT/pages/cloud/operator/add-custom-ca-to-a-pod.md

```diff
@@ -0,0 +1,188 @@
+
+## Step 3: Configure Java to load the new keystore
+
+Here we would just mount the new, modified cacerts into the default location where the JVM looks at.
+The example main uses the standard http client so alternative we could mount the cacerts to a different location and
+configure the Java runtime to load the new keystore with a system property `-Djavax.net.ssl.trustStore`.
+Note that libraries like resteasy don't respect that flag and may need to programmatically set the trust store location.
+
+```yaml
+ containers:
+   - command:
+     - /bin/bash
+     - -c
+     - |
+       curl -L https://gist.githubusercontent.com/rgolangh/b949d8617709d10ba6c690863e52f259/raw/bdea4d757a05b75935bbb57f3f05635f13927b34/Main.java -o curl.java
+       java curl.java https://kubernetes
+     image: registry.access.redhat.com/ubi9/openjdk-21
+     imagePullPolicy: Always
+     name: openjdk-21
+     volumeMounts:
+       - mountPath: /lib/jvm/java-21/lib/security
+         name: new-cacerts
+         readOnly: true
+       - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+         name: kube-api-access-5npmd
+         readOnly: true
+```
+
+Notice the volume mount of the previously mutate keystore.
+
+
+## Full working example
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: root-ca-to-cacerts
+spec:
+  initContainers:
+    - name:  add-kube-root-ca-to-cacerts
+      image: registry.access.redhat.com/ubi9/openjdk-21
+      volumeMounts:
+        - mountPath: /opt/new-cacerts
+          name: new-cacerts 
+      command:
+        - /bin/bash
+        - -c
+        - |
+          cp $JAVA_HOME/lib/security/cacerts /opt/new-cacerts/
+          chmod +w /opt/new-cacerts/cacerts
+          keytool -importcert -no-prompt -keystore /opt/new-cacerts/cacerts -storepass changeit -file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt 
+  containers:
+    - command:
+      - /bin/bash
+      - -c
+      - |
+        curl -L https://gist.githubusercontent.com/rgolangh/b949d8617709d10ba6c690863e52f259/raw/bdea4d757a05b75935bbb57f3f05635f13927b34/Main.java -o curl.java
+        java curl.java https://kubernetes
+      image: registry.access.redhat.com/ubi9/openjdk-21
+      imagePullPolicy: Always
+      name: openjdk-21
+      volumeMounts:
+      - mountPath: /lib/jvm/java-21/lib/security/
+        name: new-cacerts
+        readOnly: true
+      - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+        name: kube-api-access-5npmd
+        readOnly: true
+  volumes:
+  - name: new-cacerts
+    emptyDir: {}
+  - name: kube-api-access-5npmd
+    projected:
+      sources:
+      - serviceAccountToken:
+          path: token
+      - configMap:
+          items:
+          - key: ca.crt
+            path: ca.crt
+          name: kube-root-ca.crt
+```
+
+## Serverless Workflow Example
```

Comment:
Consider adding an example to https://github.com/apache/incubator-kie-kogito-examples/tree/main/serverless-operator-examples

Reply:
imho a new folder for this example `serverless-workflow-security-use-cases`, and under that  we add one for this case or just create one new folder for this and we can always move it around later.